README.md

```diff
@@ -0,0 +1,23 @@
+# matlab-travis-lang-test [![Build Status](https://staging.travis-ci.org/mathworks-continuous-integration/matlab-travis-lang-test.svg?branch=master)](https://staging.travis-ci.org/mathworks-continuous-integration/matlab-travis-lang-test)
+
+Repo for testing continuous integration support for MATLAB using Travis CI.
+
+## MATLAB? Using Travis CI?
+Indeed! It all starts with a minimal `.travis.yml` that looks like the following:
```

Comment:
Perhaps remove "Indeed!"?

Reply:
that looks like the following: ---> that contains this code:

Reply:
Done! Thank you 😄